ga4gh/cli.py

```diff
@@ -175,7 +197,16 @@ def __init__(self, args):
             request.callSetIds = None
         else:
             request.callSetIds = args.callSetIds.split(",")
-        setCommaSeparatedAttribute(request, args, 'variantSetIds')
+        # if no variantSets have been specified, send a request to
+        # the server to grab all variantSets and then continue
+        if args.variantSetIds is None:
+            variantSetsRequest = protocol.GASearchVariantSetsRequest()
+            response = self._httpClient.searchVariantSets(variantSetsRequest)
+            variantSets = [variantSet for variantSet in response]
```

Comment:
I'm with @Naburimannu here - we don't need the intermediate `variantSets` list do we? I.e,
```python
variantSetIds = [variantSet.id for variantSet in response]
```